setup.py

```diff
@@ -60,7 +60,7 @@
           "dogpile.cache>=0.5.3",
           "jinja2>=2.7.2",
           "lockfile>=0.9.1",
-          "pydantic[email]<2",
+          "pydantic[email]",
```

Comment:
With this change, pydantic-1 will no longer work because it doesn't have the `v1` module. That's the cause of the CI failure -- pydantic-1 is still the default on some distributions.

```suggestion
          "pydantic[email]>=2",
```

Reply:
Yes. Although, before forcing pydantic >= 2, maybe it might make sense to do some fallback magic like catching the ImportError and using the old import again, if `pydantic.v1` cannot be found? This would probably imply having to use `from pydantic.v1 import ...` style imports as we can no longer use `pydantic.v1.[ClassName]` style names.

What's the maintainers' opinion on this?

Reply:
If you want to support fallback to pydantic-1 I would accept it but personally I don't see a lot of value in continuing pydantic-1 support because I don't think there are going to be any platforms which don't have pydantic-2 available. I could be wrong though.

Reply:
Okay then, I've applied the v2 requirement now, since making the code version agnostic would be way more work. If pydantic v2 is available on most systems anyway, backwards compatibility is IMO not worth the extra work.